tests/fixtures/smoke-tests/passing-test/tests/acceptance/acceptance-test.js

```diff
@@ -0,0 +1,18 @@
+import moduleForAcceptance from '../helpers/module-for-acceptance';
+import { test } from 'qunit';
+
+let beforeEachWorks;
+
+moduleForAcceptance('Module', {
+  beforeEach() {
+    beforeEachWorks = true;
```

Comment:
@rwjblue sure, what are the arguments that I should expect?

Also, should there be assertions around `this`? What value should I expect `this` to be, or rather, what property of `this` should I expect to be non-null?